examples/resource_partitioner/shared_priority_queue_scheduler.hpp

```diff
@@ -0,0 +1,1550 @@
+//  Copyright (c) 2017-2018 John Biddiscombe
```

Comment:
Do we still need to have this as part of the examples since it is now a "real" scheduler?

Reply:
Not a must, but we left it in to have at least one example for users to look at. Since it has all the logging etc it's also a bit easier to learn from.